.travis.yml

```diff
@@ -19,26 +11,21 @@ script: bundle exec rake travis
 
 matrix:
   include:
-    - rvm: 2.1.0
-      gemfile: Gemfile
-      env: COVERAGE=true
-  exclude:
     - rvm: 1.8.7
-      gemfile: Gemfile
-    - rvm: 1.9.2
-      gemfile: Gemfile.1.8.7
-    - rvm: 1.9.3
       gemfile: Gemfile.1.8.7
-    - rvm: 2.0.0
+    - rvm: jruby-18mode
       gemfile: Gemfile.1.8.7
+    - rvm: 1.9.2
+      gemfile: Gemfile.1.9.3
```

Comment:
I believe we just dropped support to `1.9.2` at fog `2.0.0`. Could we remove it here?